config/manager/workflows/kfp/experiment-steps.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ClusterWorkflowTemplate
+metadata:
+  name: experiment-steps
+spec:
+  templates:
+  - name: create
+    metadata: {}
+    script:
+      source: |-
+        set -e -o pipefail
+        kfp-ext --endpoint '${KFP_ENDPOINT}' --output json experiment create '{{inputs.parameters.experiment-name}}' | jq -r '."ID"'
```

Comment:
This could come from a shared `kfp-operator` configmap as described in https://github.com/argoproj/argo-workflows/blob/master/examples/arguments-parameters-from-configmap.yaml